test/client/getDataFromTree.test.tsx

```diff
@@ -505,8 +505,15 @@ describe('SSR', () => {
         </ApolloProvider>
       );
 
-      return getDataFromTree(app).then(() => {
+      await getDataFromTree(app).then(html => {
+        const markup = ReactDOM.renderToStaticMarkup(app);
+        expect(markup).toEqual(html);
+        expect(markup).toMatch(/James/);
+      });
+
+      await getDataFromTree(app, ReactDOM.renderToString).then(html => {
         const markup = ReactDOM.renderToString(app);
+        expect(markup).toEqual(html);
```

Comment:
If you want to use `ReactDOM.renderToString` instead of `renderToStaticMarkup`, you can. The benefit here is that you can just call `getDataFromTree` once, without having to call `renderToString` afterwards.